From 56d04da6e553b5624ff057aaf49aeea719b69086 Mon Sep 17 00:00:00 2001
From: Marcelo Leitner <mleitner@redhat.com>
Date: Tue, 21 Oct 2014 16:20:44 -0400
Subject: [net] netfilter: nf_tables: add NFTA_MASQ_UNSPEC to nft_masq_attributes

Message-id: <20141021162044.22888.32334.stgit@localhost.localdomain>
Patchwork-id: 98004
O-Subject: [RHEL7 net PATCH 108/124] netfilter: nf_tables: add NFTA_MASQ_UNSPEC to nft_masq_attributes
Bugzilla: 1148041 1155088
RH-Acked-by: Florian Westphal <fwestpha@redhat.com>

Backport of upstream commit: 39e393bb4f653d38aea40190e1aa9a49062eed4d

 commit 39e393bb4f653d38aea40190e1aa9a49062eed4d
 Author: Pablo Neira Ayuso <pablo@netfilter.org>
 Date:   Thu Sep 11 11:02:39 2014 +0200

    netfilter: nf_tables: add NFTA_MASQ_UNSPEC to nft_masq_attributes

    To keep this consistent with other nft_*_attributes.

    Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>

Signed-off-by: Marcelo Ricardo Leitner <mleitner@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/include/uapi/linux/netfilter/nf_tables.h b/include/uapi/linux/netfilter/nf_tables.h
index eeec0ae..66d66dd 100644
--- a/include/uapi/linux/netfilter/nf_tables.h
+++ b/include/uapi/linux/netfilter/nf_tables.h
@@ -806,6 +806,7 @@ enum nft_nat_attributes {
  * @NFTA_MASQ_FLAGS: NAT flags (see NF_NAT_RANGE_* in linux/netfilter/nf_nat.h) (NLA_U32)
  */
 enum nft_masq_attributes {
+ NFTA_MASQ_UNSPEC,
  NFTA_MASQ_FLAGS,
  __NFTA_MASQ_MAX
 };
-- 
1.7.1